cypress/e2e/app/connections/connection-component/connection-component.cy.ts

```diff
@@ -0,0 +1,20 @@
+describe("Connections Connection Component", () => {
+    beforeEach(() => {
+      cy.login();
+    });
```

Comment:
Using `cy.login` method requires login between each test. Unless you specifically want to force login for each test, it may be quicker to use `cy.loginSession` instead.